app/assets/stylesheets/views/_homepage.scss

```diff
@@ -304,7 +304,7 @@
     margin-bottom: govuk-spacing(4);
 
     @include govuk-media-query($from: "tablet") {
```

Comment:
Could we swap this bit out to be from desktop so that the tablet spacing inherits the mobile spacing?
```suggestion
    @include govuk-media-query($from: "desktop") {
```

Reply:
No problem. Now updated 😎